source/user-manual/manager/manual-integration.rst

```diff
@@ -179,6 +179,41 @@ Once the configuration is complete, alerts start showing in the email inbox.
    :align: center
    :width: 80%
 
+Maltiverse
+----------
+
+`Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs) which works as a broker for Threat intelligence sources that are aggregated from more than a hundred different Public, Private and Community sources. This integration enriches any alert generated by Wazuh via the Maltiverse API, inserting new fields in case of match and following the threat taxonomy of the ECS standard (Elastic Common Squema).
+
+To set up this integration, do the following.
+
+#. Get your API key from the `Maltiverse <https://www.maltiverse.com>`__ page.
+
+#. Edit ``/var/ossec/etc/ossec.conf`` in the Wazuh server and include a configuration block such as the following. Replace ``MALTIVERSE_AUTH_TOKEN`` with your Maltiverse Authentication Token. The rule level filter is optional. You can remove it or set another level value for the integration.
+
+   .. code-block:: xml
+      :emphasize-lines: 3
+
+      <integration>
+         <name>custom-maltiverse</name>
+         <hook_url>https://api.maltiverse.com</hook_url>
+         <level>3</level>
+         <api_key>MALTIVERSE_AUTH_TOKEN</api_key>
```

Comment:
Just as a comment, the other sections add a comment where values should be replaced, like 

```
<api_key>API_KEY</api_key> <!-- Replace with your VirusTotal API key -->
```

I don't think it's necessary but perhaps we should add it to be consistent with the rest of the page